LingBuzz.js

```diff
@@ -0,0 +1,81 @@
+{
+	"translatorID": "e048e70e-8fea-43e9-ac8e-940bc3d71b0b",
+	"label": "LingBuzz",
+	"creator": "Göktuğ Kayaalp",
+	"target": "^https://ling.auf.net/lingbuzz/\\d+",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-10-07 15:14:37"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 Göktuğ Kayaalp <self at gkayaalp dot com>
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(_doc, _url) {
+	return "journalArticle";
```

Comment:
Since these are preprints, I'd save them as `report` and then add `type: article` to the Extra field so that they'll get mapped to preprint once Zotero has that available.